Mathlib/Analysis/SpecialFunctions/Pow/Real.lean

```diff
@@ -56,6 +56,11 @@ theorem rpow_intCast (x : ℝ) (n : ℤ) : x ^ (n : ℝ) = x ^ n := by
 @[simp, norm_cast]
 theorem rpow_natCast (x : ℝ) (n : ℕ) : x ^ (n : ℝ) = x ^ n := by simpa using rpow_intCast x n
 
+@[simp]
+lemma rpow_ofNat (x : ℝ) (n : ℕ) [n.AtLeastTwo] :
+    x ^ (ofNat(n) : ℝ) = x ^ (ofNat(n) : ℕ) :=
```

Comment:
Do you think it's worth having `x ^ (-ofNat(n) : ℝ) = x ^ (-ofNat(n) : \Z)` as well, as a `simp` lemma?

Reply:
Sure, why not